frontend/www/js/omegaup/typeahead.ts

```diff
@@ -252,7 +252,7 @@ export function groupTypeahead(
 ): void {
   if (!cb) {
     cb = (event: Event, val: { label: string; value: string }) =>
-      $(<EventTarget>event.target).val(val.label);
+      $(<EventTarget>event.target).attr('data-alias', val.value);
```

Comment:
```suggestion
      $(<EventTarget>event.target)
          .attr('data-alias', val.value)
          .val(val.label);
```

esto todavía es necesario, porque hay otros lugares donde se usa y hay que preservar el comportamiento.

Reply:
Listo